### DISCUSSION TITLE
Datapusher API key

### DISCUSSION THREAD

**Discussion by vpvasso** [ckan/ckan#7782] [Q&A]
Hello everyone,
I'm new here and i need some help. I saw that starting from CKAN 2.10, DataPusher requires a valid API token to operate, and will fail to start if the following option is not set:
ckan.datapusher.api_token = <api_token>
Can you help me to get my Datapusher API key?

**Comment by hongjie123** [comment]
Before configuring the datapusher, you need to create a system administrator (reference: https://docs.ckan.org/en/2.10/maintaining/getting-started.html#creating -A-sysadmin user, then log in to your site and add an API key to the user's personal center.
